app/models/concerns/user_authorisation.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module UserAuthorisation
+  extend ActiveSupport::Concern
+
+  included do
+    def current_role
+      # this is awful
+      establishment_users.find_by({ establishment_id: establishment.id }).role
```

Comment:
pourquoi "this is awful" ?

Reply:
😄 c'est juste pas limpide comme code ; j'ai tendance à faire du over/understatement 🇬🇧